theano/tensor/tests/test_opt.py

```diff
@@ -6122,6 +6123,38 @@ def test3(self):
         assert len(divs) == 0
 
 
+def test_local_expm1():
+    x = matrix('x')
+    u = T.scalar('u')
+
+    y = T.exp(x) - 1.
+    z = T.exp(x) - 2.
+    t = T.exp(x) - x
+    s = T.exp(u) - numpy.ones((4, 3)).astype(config.floatX)
+
+    f = function([x], y)
+    g = function([x], z)
+    h = function([x], t)
+    r = function([u], s)
+    x_val = numpy.random.rand(4, 3).astype(config.floatX)
+    f_val = f(x_val)
+    f_test = function([x], T.expm1(x))
```

Comment:
pass mode=theano.compile.get_default_mode().including('local_expm1')

It should fix travis.